plugin/src/shared/sc/plugin2021/xstream/BoardConverter.kt

```diff
@@ -5,11 +5,16 @@ import com.thoughtworks.xstream.converters.MarshallingContext
 import com.thoughtworks.xstream.converters.UnmarshallingContext
 import com.thoughtworks.xstream.io.HierarchicalStreamReader
 import com.thoughtworks.xstream.io.HierarchicalStreamWriter
+import org.slf4j.LoggerFactory
 import sc.plugin2021.util.Constants
 import sc.plugin2021.Board
 import sc.plugin2021.FieldContent
 
 class BoardConverter: Converter {
+    companion object {
```

Comment:
use klogging

Reply:
Isn't this what's being used everywhere else?
Doesn't it make more sense to have just one system everywhere?
Or doesn't it matter anyways?

Reply:
oh, I thought we already used klogging widely, I guess not.